lib/client.js

```diff
@@ -42,6 +42,21 @@ function Client(dispatcher, options) {
    * @type {Projects}
    */
   this.projects = new resources.Projects(this.dispatcher);
+  /**
+   * An instance of the ProjectMemberships resource.
+   * @type {ProjectMemberships}
+   */
+  this.projectsMemberships = new resources.ProjectMemberships(this.dispatcher);
```

Comment:
I hadn't noticed this before if this is present in the docs, but shouldn't this be "projectMemberships"? etc.